packages/lit-html/src/static.ts

```diff
@@ -150,15 +180,25 @@ export const withStatic =
     }
 
     if (hasStatics) {
+      let innerCache = stringsCache.get(strings);
+      if (innerCache === undefined) {
+        if (isServer) {
+          innerCache = new LRUCache<string, TemplateStringsArray>(10);
+        } else {
+          innerCache = new Map<string, TemplateStringsArray>();
+        }
```

Comment:
Code style nitpick; Feel free to ignore.
```suggestion
        innerCache = isServer
          ? new LRUCache<string, TemplateStringsArray>(10)
          : new Map<string, TemplateStringsArray>();
```